specs/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@instantsearch/specs-website",
+  "version": "0.1.0",
+  "private": true,
+  "description": "The specs website for InstantSearch.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/algolia/instantsearch.js"
+  },
+  "license": "MIT",
+  "author": "Algolia <support@algolia.com>",
+  "scripts": {
+    "build": "astro build",
+    "check": "astro check && tsc",
+    "dev": "astro dev",
+    "preview": "astro preview",
+    "start": "astro dev",
+    "prepare": "rm -rf public && mkdir public && cp -r ../node_modules/instantsearch.css/themes public/themes"
```

Comment:
wonder if this can be done with node_modules directly without the prepare script, but can be done after merge (ore never lol)

Reply:
In Astro you need `<link>` tags to reference either an absolute URL or a relative from the `public/` directory, so it's difficult. Imports would fix this but it's difficult to manage switching stylesheets with static imports and the side-effect nature of CSS.